AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -159,6 +191,7 @@ - (ASLayout *)filteredNodeLayoutTree
     if (self != context.layout && context.layout.type == ASLayoutElementTypeDisplayNode) {
       ASLayout *layout = [ASLayout layoutWithLayout:context.layout position:context.absolutePosition];
       layout.flattened = YES;
+      layout.cacheSublayoutLayoutElements = YES;
```

Comment:
I'm not sure why we need this. Retaining "grand" subnodes should be the job of each subnode, no?